### PR TITLE
chore(html) add spreadsheet helpers

### PR DESCRIPTION
**Pull request by zhpenkov**
Adds helpers for the **Spreadsheet** component as well as the **Tabstrip** and **Menu** components used inside.

### CODE REVIEW

File: tests/tabstrip/tabstrip.html [hunk 1]
```diff
@@ -25,38 +25,38 @@
...
+                                    <li class="k-tab-on-top k-item k-active k-first">
                                         <span class="k-link">Inner Tab 1</span>
                                     </li>
                                     <li class="k-item k-last">
                                         <span class="k-link">Inner Tab 2</span>
                                     </li>
                                 </ul>
                             </div>
-                            <div class="k-tabstrip-content k-active" style="display: block; margin-left: 128px; min-height: 68px;">Left orientation inside Top orientation</div>
+                            <div class="k-tabstrip-content" style="display: block; margin-left: 128px; min-height: 68px;">Left orientation inside Top orientation</div>
```
Comment (Juveniel) [inline tests/tabstrip/tabstrip.html:46]: should we bring back k-active here ?

File: tests/spreadsheet/spreadsheet.html [hunk 2]
```diff
@@ -338,7 +338,7 @@
                                 <div class="k-vertical-align-center">Amount</div>
                             </div>
                             <div class="k-spreadsheet-cell" style="background: rgb(167, 214, 255); outline: rgb(150, 193, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 816px; top: 71px; width: 63px; height: 24px;"></div>
-                            <div class="k-spreadsheet-cell k-spreadsheet-active-cell k-left k-single" style="background: rgb(255, 255, 255); outline: rgb(230, 230, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 1px; top: 96px; width: 99px; height: 29px;">
+                            <div class="k-spreadsheet-cell k-left k-single k-spreadsheet-active-cell" style="background: rgb(255, 255, 255); outline: rgb(230, 230, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 1px; top: 96px; width: 99px; height: 30px;">
```
Comment (Juveniel) [inline tests/spreadsheet/spreadsheet.html:341]: the height difference probably causes the misalignment of the selection in the related test

File: tests/spreadsheet/spreadsheet-tools.html [hunk 3]
```diff
@@ -303,29 +303,20 @@
...
-                            </span>
-                        </button>
-                    </div>
+                    <button class="k-toolbar-button  k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                        <span class="k-button-icon k-icon k-svg-icon k-svg-i-text-wrap">
+                            <svg aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                                <path d="M480 96v32H96V96zM352 256v224H32V256zm-32 32H64v160h256zm-32 32H96v32h192zm0 96v-32H96v32zM64 64h288V32H32v160h320v-32H64z"></path>
+                            </svg>
+                        </span>
+                    </button>
```
Comment (Juveniel) [inline tests/spreadsheet/spreadsheet-tools.html:312]: is this change intentional ?

File: tests/_output/default/tabstrip/tabstrip-rtl.png
Comment (Juveniel) [inline tests/_output/default/tabstrip/tabstrip-rtl.png:1]: dragging state is missing in left/right orientation.

File: tests/_output/default/spreadsheet/spreadsheet.png
Comment (Juveniel) [inline tests/_output/default/spreadsheet/spreadsheet.png:1]: border width of active cell is not correct

File: packages/html/src/menu/tests/menu-vertical.tsx [hunk 4]
```diff
@@ -86,12 +87,12 @@ export default () =>(
...
                     </li>
-                </ul>
+                </Menu>
             </section>
 
             <section>
                 <div className="k-menu-scroll-wrapper vertical">
-                    <ul id="menu" className="k-widget k-reset k-header k-menu k-menu-vertical" style={{ height: "150px" }} >
+                    <Menu orientation="vertical" style={{ height: "150px" }} >
                         <li className="k-item k-menu-item k-first">
```
Comment (Juveniel) [inline packages/html/src/menu/tests/menu-vertical.tsx:96]: We should re-use MenuItem here

**Review by Juveniel — COMMENTED** [review]
Need a few more tweaks, but looks great overall!

File: tests/_output/default/tabstrip/tabstrip-rtl.png
Comment (zhpenkov) [reply tests/_output/default/tabstrip/tabstrip-rtl.png:1]: LTR orientation test ( **tabstrip.tsx** )  had a dragging state before I added one in ( **tabstrip-rtl** ).
Tests are now alike, and only differ by **dir**.
[tabstrip.png](https://github.com/telerik/kendo-themes/blob/develop/tests/_output/default/tabstrip/tabstrip.png)

**Review by Juveniel — APPROVED** [review]
